tests.py

```diff
@@ -245,11 +245,11 @@ def test_noise_sfts(self):
         # peak freqs expected exactly equal to first case,
         # peak values can have a bit of numerical diff
         self.assertTrue(np.all(max_freqs_added_signal == max_freqs_noise_and_signal))
-        self.assertTrue(
-            np.allclose(
-                max_values_added_signal, max_values_noise_and_signal, rtol=1e-6, atol=0
-            )
-        )
+        # self.assertTrue(
+        #    np.allclose(
+        #        max_values_added_signal, max_values_noise_and_signal, rtol=1e-6, atol=0
+        #    )
+        # )
```

Comment:
Regarding the actual value of the peaks, we are somehow digging our own grave (again): Noise is random, that doesn't have to be true.

I understand that the relative difference should "generally" between well understood intervals, so I'm more willing to make this snippet work.

Reply:
This part explicitly compares the two paths of "create noise and signal together" vs "first create noise, then inject signal" with the same random seed for the noise on both paths. So I think this one actually should always be able to pass at good precision.

Reply:
I agree: Nuke the "unique frequency peak", make this one work.

Reply:
Interesting, if I use `rtol=1e-6` and print relevant quantities to [numpy.allclose](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html):
```
print(f"Absolute diff: {max_values_added_signal-max_values_noise_and_signal} , rtols: {1e-6*max_values_added_signal}, {1e-6 * max_values_noise_and_signal}")

Absolute diff: [  7.199463    8.605072  -18.658844   -1.1877441] , rtols: [0.00051476 0.00016379 0.00031445 0.00068457], [0.00050756 0.00015519 0.00033311 0.00068576]
```

Reply:
And it persists if we just scale `SqrtSX` and `h0` down to `1e-22`:
```
Absolute diff: [ 7.1995086e-22  8.6050694e-22 -1.8658836e-21 -1.1877797e-22] , rtols: [5.1476252e-26 1.6379310e-26 3.1444997e-26 6.8456936e-26], [5.0756303e-26 1.5518803e-26 3.3310881e-26 6.8575715e-26]
```

Reply:
Solved in https://github.com/PyFstat/PyFstat/pull/207#issuecomment-705555667